### PR TITLE
Completed task 5. GET /api/articles/:article_id

### PR DESCRIPTION
**Pull request by juliaozm**
Submitting request for task 5

### CODE REVIEW

File: db/app.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
 const express = require('express')
```
Comment (xViolaine) [inline db/app.js]: Generally speaking, your app.js should live outside of the db folder

File: __tests__/app.test.js [hunk 2]
```diff
@@ -40,4 +41,91 @@ describe('news-project', () => {
...
+               
+            })
+        })
+
+        test('GET: 404 - a get request should return a message "Not Found" when "article_id" does not exist', () => {
+            return request(app).get('/api/articles/100').expect(404)
+            .then(({body : {message}}) => {
+                expect(message).toEqual('Not Found')
+            })
+        })
```
Comment (xViolaine) [inline __tests__/app.test.js:128]: We also want to make sure to include a test for a 400 for when a ID is used that will never exist

**Review by xViolaine — CHANGES_REQUESTED** [review]
Good job, just an extra test needed and a small change in the file structure!

**Review by Hard-Luck — APPROVED** [review]
Good work, good to merge!

**Pull request merged**